File: Java Code/Kaleidoscope.java
package Challenge;

import java.util.Scanner;

public class Kaleidoscope {
    public static void main(String[] args){
        final double TAX=1.07;
        final double DISCOUNT=0.1;
        final double PRICE=5.0;
        Scanner sc = new Scanner(System.in);
        int num=sc.nextInt();
        double sum =num *PRICE;
        if (num>1) sum*= (1-DISCOUNT);
        sum *= TAX;
        // System.out.println(sum);
        System.out.format("%.2f",sum);
        sc.close();
    }
}

File: C and CPP/Kaleidoscope.cpp
#include <iostream>
using namespace std;
int main(){
    const double TAX=0.07;
    const double DISCOUNT=0.1;
    const double PRICE=5.0;
    int amount;
    cin>>amount;
    double sum= amount * PRICE;
    if (amount >1 ) sum *= (1-DISCOUNT);
    sum *= (1+TAX);
    // cout<<sum<<endl;
    printf("%.2f",sum);
}